tmt/cli.py

```diff
@@ -1750,9 +1750,21 @@ def status(
 # inference. See Context and ContextObjects above.
 @main.group(chain=True, invoke_without_command=True, cls=CustomGroup)  # type: ignore[arg-type]
 @pass_context
+@option(
+    '-l', '--last', is_flag=True, help='Clean everything of the last run.')
+@option(
+    '-i', '--id', 'id_', metavar="ID",
+    help='Run id (name or directory path) to clean everything of.')
+@option(
+    '-s', '--skip', type=str, default=None,
+    help='The clean phases to skip')
 @verbosity_options
 @dry_options
-def clean(context: Context, **kwargs: Any) -> None:
+def clean(context: Context,
+          last: bool,
+          id_: Optional[str],
+          skip: Optional[str],
```

Comment:
With `skip` becoming a `multiple=True`, this would be a `list[str]`.

Reply:
Already addressed.